### PR TITLE
Fixes for IG benchmarks. Added token path var, changed how test works.

### PR DESCRIPTION
**Pull request by Atomic182**
- Changed init(Token generation simulation) sim to create as many tokens as there is users
- Path to token file is now variable
- Cleaned up tests a bit.

### CODE REVIEW

File: 6.5/benchmarks/gatling-simulation-files/ig/IGGenerateTokensSim.scala [hunk 1]
```diff
@@ -59,18 +53,16 @@ class IGGenerateTokens extends Simulation {
...
+        exec(
           http("Request Token stage")
             .post("/oauth2/access_token")
-            .queryParam("realm", realm)
+             .queryParam("realm", realm)
             .formParam("grant_type", "password")
-            .formParam("scope", scope)
-            .formParam("username", "${username}")
-            .formParam("password", "${password}")
+            .formParam("scope", "cn")
```
Comment (Atomic182) [inline 6.5/benchmarks/gatling-simulation-files/ig/IGGenerateTokensSim.scala]: Ah, wrong scope, will change back

File: 6.5/benchmarks/gatling-simulation-files/ig/IGAccessTokensSim.scala [hunk 2]
```diff
@@ -1,30 +1,24 @@
...
 import io.gatling.http.protocol.HttpProtocolBuilder
 
-class IGAccessTokenSim extends Simulation {
+class IGAccessTokensSim extends Simulation {
 
   val concurrency: Integer = Integer.getInteger("concurrency", 10)
   val duration: Integer = Integer.getInteger("duration", 600)
   val warmup: Integer = Integer.getInteger("warmup", 1)
-  val igHost: String = System.getProperty("ig_host", "openig.prod.perf.forgerock-qa.com")
+  val igHost: String = System.getProperty("ig_host", "login.prod.perf.forgerock-qa.com")
```
Comment (lee-baines) [inline 6.5/benchmarks/gatling-simulation-files/ig/IGAccessTokensSim.scala]: why is this changed to am url?

Comment (Atomic182) [reply 6.5/benchmarks/gatling-simulation-files/ig/IGAccessTokensSim.scala]: My bad, will change it.
